docs/proposals/maintenance-mode.md

```diff
@@ -0,0 +1,43 @@
+# Proposal: Node Maintenance Mode (MM)
+
+Last updated: 8.10.2021
+
+
+## Abstract
+
+Node Maintenance is a removing node temporarily. All CSI components, except DaemonSets (csi-baremetal-node, csi-baremetal-se), will be delete by the CSI Operator.
+
+## Background
+
+A user might want to take a single node down for maintenance (say to repair a bad hardware component) or they might do a sequential rolling maintenance mode (MM) workflow (say to run software upgrade on all nodes). In the rolling MM case one node enters MM, finishes maintenance operation (say software upgrade), then is exited from MM and immediately the next node is put in MM without any time in between.
```

Comment:
I don't think that we need to provide examples here (especially using word `say`). Just explain how CSI will be handling procedure and why some pods aren't removed